src/wp-includes/author-template.php

```diff
@@ -493,7 +510,8 @@ function wp_list_authors( $args = '' ) {
 			$link .= '>';
 
 			if ( ! empty( $args['feed_image'] ) ) {
-				$link .= '<img src="' . esc_url( $args['feed_image'] ) . '" style="border: none;"' . $alt . ' />';
+				$attr = cp_attributes( 'img', array( 'src' => $args['feed_image'], 'alt' => $alt, 'style' => 'border: none;' ) );
```

Comment:
this one makes the `alt` exist regardless of the feed parameter, instead of leaving it off if empty
the test does not expect it to be there if empty